Updating compatibility and events tables (#7973)
magento/devdocs: src/recommendations/events.md

`add-to-wishlist` | product | A product is added to the wishlist
`apply-discount-action` | shopping-cart | A discount code is applied to the cart
`edit-product-qty` | shopping-cart | The quantity of a product in a cart changed to a non-zero value
`impression-render` | recommendation-unit | The recommendation unit is rendered on the page.
`initiate-checkout` | shopping-cart | The customer initiates checkout
`instant-purchase` | checkout |  The customer completes an instant purchase
`place-order` | checkout | The customer's order is submitted
`rec-add-to-cart-click` | recommendation-unit | The customer clicks the **Add to cart** button for an item in the recommendation unit.
`rec-click` | recommendation-unit | The customer clicks a product in the recommendation unit.
`remove-discount-action` | shopping-cart | A discount code is removed from a cart
`remove-from-cart` | product | A product is removed from a cart
`remove-from-wishlist` | product | A product is removed from a wishlist
`sign-in` | shopper | The customer signs into their account
`sign-out` | shopper | The customer signs out of their account
`view` | product | The customer views a product details page (PDP)

> `view` | recommendation-unit | The recommendation unit becomes viewable in the shopper's browser, such as by scrolling into view, appearing, and so on

Reviewer: "Appearing" doesn't seem right. I recommend ending the line at "into view".

Since you have two `view` events, you might want to consider having a separate table for each category. I guess it depends on how many events and categories you expect to add later on.

Neither of my suggestions here are critical. Approved.

Author: Good point about two tables. I'll keep them all within a single table for now and break out the future if the list expands.

{:.bs-callout-info}
If [Cookie Restriction Mode](https://docs.magento.com/m2/ce/user_guide/stores/compliance-cookie-restriction-mode.html) is enabled, Magento does not collect behavioral data until the shopper consents. If Cookie Restriction Mode is disabled, Magento collects behavioral data by default.

Event | Category | Description
--- | ---
`add-to-cart` | product | A product is added to the cart
`add-to-wishlist` | product | A product is added to the wishlist
`apply-discount-action` | shopping-cart | A discount code is applied to the cart
`edit-product-qty` | shopping-cart | The quantity of a product in a cart changed to a non-zero value

> `impression-render` | recommendation-unit | The recommendation unit is rendered on the page. If there are _N_ items rendered, there will be _N_ recommended item contexts attached to this event (one per item recommended).

Reviewer 2: ```suggestion
`impression-render` | recommendation-unit | The recommendation unit is rendered on the page.
```
I would focus on describing the trigger for this event

`initiate-checkout` | shopping-cart | The customer initiates checkout
`instant-purchase` | checkout |  The customer completes an instant purchase
`place-order` | checkout | The customer's order is submitted

> `rec-add-to-cart-click` | recommendation-unit | The customer clicks the **Add to cart** button for an item in the recommendation unit. The recommended item context represents the item the customer clicked.

Reviewer 2: ```suggestion
`rec-add-to-cart-click` | recommendation-unit | The customer clicks the **Add to cart** button for an item in the recommendation unit.
```

> `rec-click` | recommendation-unit | **Not supported on Chrome**. The customer clicks a product in the recommendation unit. The recommended item context represents the item the customer clicked.

Reviewer 2: ```suggestion
`rec-click` | recommendation-unit | The customer clicks a product in the recommendation unit.
```
"not supported in Chrome" would send the wrong message here. let's keep it simple

`remove-discount-action` | shopping-cart | A discount code is removed from a cart
`remove-from-cart` | product | A product is removed from a cart
`remove-from-wishlist` | product | A product is removed from a wishlist
`sign-in` | shopper | The customer signs into their account
`sign-out` | shopper | The customer signs out of their account
`view` | product | The customer views a product details page (PDP)

> `view` | recommendation-unit | The recommendation unit becomes viewable on the page, such as by scrolling into view, appearing, and so on

Reviewer 2: ```suggestion
`view` | recommendation-unit | The recommendation unit becomes viewable in the shopper's browser, such as by scrolling into view, appearing, and so on
```

Reviewer 2: also, can we include very high level summary of this information on this page as well? (can treat as separte task if you want): 
https://support.liftigniter.com/support/solutions/articles/30000027124-liftigniter-and-ad-blockers

mainly: some shoppers will use adblockers or privacy settings that will prevent our systems from seeing and capturing the events, as well as routine network packet loss, etc. therefore the numbers you'll see reflected in metrics will never be exact, including generated revenue. if anything, the engagement and revenue numbers will be underrepresented.
